### ISSUE TITLE
Rosalind.info problems

### ISSUE THREAD

**Issue by kescobo** [BioJulia/BioTutorials#8]
One place to find useful things to demo is [rosalind.info](https://rosalind.info/problems/list-view/), which has a lot of bioinformatics problems.

One idea would be to show how to do each thing naively with strings, and then do it again "the right way" with BioJulia packages.

## Problems

- [x] [base composition](https://rosalind.info/problems/dna/) #9
- [x] [transcription](https://rosalind.info/problems/rna/) #9 
- [x] [reverse complement](https://rosalind.info/problems/revc/) #9 
- [x] [fibbonacci](https://rosalind.info/problems/fib/) #14
- [x] [gc content](https://rosalind.info/problems/gc/)  #14 
- [x] [hamming distance](https://rosalind.info/problems/hamm/) #15 
- [ ] [mendel's first law](https://rosalind.info/problems/iprb/) #16

**Comment by danielle-pinto** [comment]
Yeah, I noticed that. I tagged you in some comments on the last PR  that was merged in. If you have thoughts, great. I'm looking into it! I was a bit surprised, since all that changed was the markdown file and the vitepress version.
